mkdocs/getting_started.md

```diff
@@ -330,11 +330,34 @@ In order to complete this tutorial you'll one of these devices:
 
 !!! question
 
-    Don't know which serial port to use? Use this guide [Find Arduino Port
-    on Windows, Mac, and
-    Linux](https://www.mathworks.com/help/supportpkg/arduinoio/ug/find-arduino-port-on-windows-mac-and-linux.html)
-    from the MATLAB docs to help. Simply ignore that its made for Arduino, this
-    guide will work for any serial USB device.
+    Don't know which serial port to use?
+    ### On Linux
+    With the device unplugged, run the below command
+    ```
+    $ ls /dev/ttyUSB*
```

Comment:
Serial Devices can also have the name /dev/ttyACM0. So we need to consider that as well.

Reply:
added a section for ACM0 under the linux section